src/style.css

```diff
@@ -0,0 +1,30 @@
+.main-content {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+/*  s*/
+.stars__item {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__item:last-child {
+  margin-right: 0;
+}
```

Comment:
You can set `margin-right: 4px;` here and make a use of :not pseudo class to exclude last child